pkg/handler/handler.go

```diff
@@ -720,7 +720,7 @@ func (h handler) lastoperation(w http.ResponseWriter, r *http.Request, params ma
 		if err != nil {
 			switch err {
 			case broker.ErrorNotFound:
-				writeResponse(w, http.StatusGone, nil)
+				writeResponse(w, http.StatusGone, make(map[int]bool, 1))
```

Comment:
This is probably pedantic, but the canonical golang json representation is `map[string]interface{}`. https://blog.golang.org/json-and-go.

Reply:
That's exactly what I needed. Thanks!